packages/path_provider/path_provider/lib/path_provider.dart

```diff
@@ -202,17 +128,12 @@ enum StorageDirectory {
 /// On Android this returns Context.getExternalFilesDirs(String type) or
 /// Context.getExternalFilesDir(String type) on API levels below 19.
 Future<List<Directory>> getExternalStorageDirectories({
-  /// Optional parameter. See [StorageDirectory] for more informations on
+  /// Optional parameter. See [AndroidStorageDirectory] for more informations on
   /// how this type translates to Android storage directories.
-  StorageDirectory type,
+  AndroidStorageDirectory type,
```

Comment:
**Warning**: I think renaming this Enum is a backwards incompatible change; people who were using `StorageDirectory` in their apps will now have to migrate them to `AndroidStorageDirectory`... I think you should keep the `StorageDirectory` enum name, even if it only applies for Android.

(However, `macos` has a `downloads` directory, maybe the path provider will be able to access it? :P)